src/config/dark-sites.config

```diff
@@ -727,6 +727,7 @@ sumo.app
 sunxdcc.com
 supabase.io
 supinic.com
+support.steampowered.com
```

Comment:
Support? I'm getting a redirect to help.steampowered.com

Reply:
idk the actual help pages seem to still be on the support subdomain eg:
https://support.steampowered.com/kb_article.php?ref=5414-tfbn-1352
https://support.steampowered.com/kb_article.php?ref=7480-WUSF-3601
https://support.steampowered.com/kb_article.php?ref=4044-QDHJ-5691

Reply:
well help.steampowered.com is dark too lol